Resources/Textures/Structures/Storage/Crates/command.rsi/meta.json

```
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from Citadelstation at commit https://github.com/Citadel-Station-13/Citadel-Station-13/commit/85186a971453b0653bacfb3fae88f978dc1be689",
+  "copyright": "Taken from Citadelstation at commit https://github.com/Citadel-Station-13/Citadel-Station-13/commit/85186a971453b0653bacfb3fae88f978dc1be689, resprite by mureixlol",
```

От оригинала ничего не осталось, просто укажи:
```suggestion
  "copyright": Made by Discord: @mureixlol",
```